BankAPILibrary/GiniBankAPILibraryExample/GiniBankAPILibraryExampleTests/ExtractionFeedbackIntegrationTest.swift

```diff
@@ -0,0 +1,118 @@
+//
+//  ExtractionFeedbackIntegrationTest.swift
+//  GiniBankAPILibraryExampleTests
+//
+//  Created by Nadya Karaban on 04.03.22.
+//
+
+import Foundation
+
+import XCTest
+@testable import GiniBankAPILibrary
+
+class ExtractionFeedbackIntegrationTest: XCTestCase {
+    let clientId = ProcessInfo.processInfo.environment["CLIENT_ID"]!
+    let clientSecret = ProcessInfo.processInfo.environment["CLIENT_SECRET"]!
+    var giniBankAPILib: GiniBankAPI!
+    var documentService: DefaultDocumentService!
+    
+    override func setUp() {
+        giniBankAPILib = GiniBankAPI
+               .Builder(client: Client(id: clientId,
+                                       secret: clientSecret,
+                                       domain: "pay-api-lib-example"))
+               .build()
+        documentService = giniBankAPILib.documentService()
+    }
+    func loadFile(withName name: String, ofType type: String) -> Data {
+        let fileURLPath: String? = Bundle.main
+            .path(forResource: name, ofType: type)
+        let data = try? Data.init(contentsOf: URL(fileURLWithPath: fileURLPath!))
+        
+        return data!
+    }
+    
+    func testSendExtractionFeedback() {
+
```

Comment:
The other function bodies don't start with empty line. I'm fine both with or without the empty line. Just keep it consistent. 🤓

Reply:
👌🏻I'll fix it